### PR TITLE
[RW-5476][risk=no] WDL for RandomizeVcf

### PR DESCRIPTION
**Pull request by als364**
This PR has a wdl for RandomizeVcf plus a markdown file with instructions for running it on Terra.

---
**PR checklist**

- [x] This PR meets the Acceptance Criteria in the JIRA story
- [x] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [x] I have run and tested this change locally
- [ ] I have run the E2E tests on this change against my local UI + API server with `yarn test:local`
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: api/genomics/wdl/RandomizeVcf/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# To build and deploy, from this directory:
```
Comment (ericsong) [inline api/genomics/wdl/RandomizeVcf/Dockerfile:1]: We can probably get rid of most of these comments as they're related to Circle

File: api/genomics/wdl/IngestSyntheticVcfWf.wdl [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+    /workbench/api/project.rb randomize-vcf \
+    --vcf ~{base_vcf} \
+    --sample-names-file ~{file_of_sample_names} \
+    --output $(pwd)/~{output_file}
+
+  >>>
+
+  runtime {
+    memory: "4 GiB"
```
Comment (ericsong) [inline api/genomics/wdl/IngestSyntheticVcfWf.wdl:75]: should we specify CPU as well?

**Review by ericsong — APPROVED** [review]

File: api/genomics/wdl/IngestSyntheticVcfWf.wdl (hunk 2 above)
Comment (als364) [reply api/genomics/wdl/IngestSyntheticVcfWf.wdl:75]: Yeah, probably should

File: api/genomics/wdl/RandomizeVcf/Dockerfile (hunk 1 above)
Comment (als364) [reply api/genomics/wdl/RandomizeVcf/Dockerfile:1]: 👍

File: api/genomics/wdl/IngestSyntheticVcfWf.wdl (hunk 2 above)
Comment (als364) [reply api/genomics/wdl/IngestSyntheticVcfWf.wdl:75]: Actually, yeah, I'll just turn this into an n2-standard-2

**Comment by als364** [comment]
puppeteer test is again a timeout in a notebook test

**Comment by als364** [comment]
now it's other more different timeouts. merging

**Pull request merged**